api/utils/requestProcessor.js

```diff
@@ -697,6 +697,17 @@ const processRequest = (params) => {
                         });
                     });
                     break;
+                case 'stop':
+                    validateUserForWrite(params, () => {
+                        taskmanager.stopTask({
+                            db: common.db,
```

Comment:
You are passing there task_id and operation id.
And as I check taskmaager.stopTask() you are killing operation based on operation id. In that moment you are not checking if it is connected to any task.

So it means if in some evil way I get global admin API key  - I can kill any operation as long as I guess operation id.

I would suggest rewriting inside stopTask a bit.
Can we do it similar like with rerun Task?

1)Check if task exists
https://github.com/Countly/countly-server/blob/master/api/utils/taskmanager.js#L827
2)And only then kill something. And kill based on operation id you read from mongo. Not the one you recieved.